ui/src/treeDataProvider/AzExtParentTreeItem.ts

```diff
@@ -248,10 +248,7 @@ export abstract class AzExtParentTreeItem extends AzExtTreeItem implements types
     }
 
     private async loadMoreChildrenInternal(context: types.IActionContext): Promise<void> {
-        this._isLoadingMore = true;
         try {
-            this.treeDataProvider.refreshUIOnly(this);
```

Comment:
I moved this to `AzExtTreeDataProvider.loadMore` because it was causing too many refreshes to happen, making it impossible to expand an InvalidTreeItem. It has to do with the fact that VS Code saves collabsible state based on a tree item's id and we always use a [unique id](https://github.com/microsoft/vscode-azuretools/blob/master/ui/src/treeDataProvider/AzExtParentTreeItem.ts#L374) for InvalidTreeItems.

`loadMoreChildrenInternal` is called when a user clicks "Load more..." _and_ the first time a tree item is expanded, whereas `AzExtTreeDataProvider.loadMore` is _only_ called when a user clicks "Load more...".